src/test/java/reposense/parser/StandaloneConfigJsonParserTest.java

```diff
@@ -0,0 +1,107 @@
+
+    private static RepoConfiguration EXPECTED_GITHUBID_ONLY_REPOCONFIG;
+    private static RepoConfiguration EXPECTED_FULL_REPOCONFIG;
+
+    @BeforeClass
+    public static void setUp() throws InvalidLocationException {
+        Author author = new Author("yong24s");
+        author.setAuthorAliases(Arrays.asList("Yong Hao TENG"));
+        author.setIgnoreGlobList(Arrays.asList("**.css", "**.html", "**.jade", "**.js"));
+
+        EXPECTED_GITHUBID_ONLY_REPOCONFIG = new RepoConfiguration(TEST_DUMMY_LOCATION);
+        EXPECTED_GITHUBID_ONLY_REPOCONFIG.setFormats(ArgsParser.DEFAULT_FORMATS);
+        EXPECTED_GITHUBID_ONLY_REPOCONFIG.setAuthorList(Arrays.asList(new Author("yong24s")));
+
+        EXPECTED_FULL_REPOCONFIG = new RepoConfiguration(TEST_DUMMY_LOCATION);
+        EXPECTED_FULL_REPOCONFIG.setFormats(Arrays.asList("gradle", "jade", "java", "js", "md", "scss", "yml"));
+        EXPECTED_FULL_REPOCONFIG.setIgnoreCommitList(Arrays.asList("7b96c563eb2d3612aa5275364333664a18f01491"));
+        EXPECTED_FULL_REPOCONFIG.setIgnoreGlobList(Arrays.asList("**.adoc", "collate**"));
+        EXPECTED_FULL_REPOCONFIG.setAuthorList(Arrays.asList(author));
+        EXPECTED_FULL_REPOCONFIG.setAuthorDisplayName(author, "Yong Hao");
+        EXPECTED_FULL_REPOCONFIG.addAuthorAliases(author, Arrays.asList(author.getGitId()));
+        EXPECTED_FULL_REPOCONFIG.addAuthorAliases(author, author.getAuthorAliases());
+    }
+
+    @Test
+    public void standaloneConfig_parseEmptyTextFile_success() throws IOException {
+        new StandaloneConfigJsonParser().parse(STANDALONE_CONFIG_EMPTY_TEXT_FILE);
+    }
+
+    @Test
+    public void standaloneConfig_parseEmptyJsonFile_success() throws IOException {
+        new StandaloneConfigJsonParser().parse(STANDALONE_CONFIG_EMPTY_JSON_FILE);
+    }
+
+    @Test
+    public void standaloneConfig_ignoresUnknownProperty_success() throws IOException {
+        new StandaloneConfigJsonParser().parse(STANDALONE_UNKNOWN_PROPERTY_CONFIG);
+    }
+
+    @Test
+    public void standaloneConfig_correctConfig_success() throws IOException, InvalidLocationException {
+        StandaloneConfig config = new StandaloneConfigJsonParser().parse(STANDALONE_CONFIG_FULL);
+        isExact(EXPECTED_FULL_REPOCONFIG, config);
+    }
+
+    @Test
+    public void standaloneConfig_githubIdOnlyConfig_success() throws IOException, InvalidLocationException {
+        StandaloneConfig config = new StandaloneConfigJsonParser().parse(STANDALONE_CONFIG_GITHUBID_ONLY);
+        isExact(EXPECTED_GITHUBID_ONLY_REPOCONFIG, config);
+    }
+
+    @Test(expected = JsonSyntaxException.class)
+    public void standaloneConfig_malformedJsonFile_throwsIoException() throws IOException {
+        new StandaloneConfigJsonParser().parse(STANDALONE_MALFORMED_CONFIG);
+    }
+
+    private void isExact(RepoConfiguration expectedRepoConfig, StandaloneConfig actualStandaloneConfig)
```

Comment:
Not sure if `isExact` sounds correct because exact usually refers to 2 reference pointing to the same object.
Also, since this method does not return a boolean result, we probably shouldn't prefix it with an `is`.

Perhaps `assertSame` would be better to imply the duplication and internal assertion.